tests/pivots/spreadsheet_pivot/spreadsheet_pivot_side_panel.test.ts

```diff
@@ -314,6 +314,29 @@ describe("Spreadsheet pivot side panel", () => {
     expect(fixture.querySelectorAll(".pivot-dimension")).toHaveLength(0);
   });
 
+  test("filter unsupported measures", async () => {
+    setCellContent(model, "A1", "integer");
+    setCellContent(model, "A2", "10");
+    setCellContent(model, "B1", "float");
+    setCellContent(model, "B2", "10.1");
+    setCellContent(model, "C1", "bool");
+    setCellContent(model, "C2", "true");
+    setCellContent(model, "D1", "date");
+    setCellContent(model, "D2", "2024/01/01");
+    setCellContent(model, "E1", "datetime");
+    setCellContent(model, "E2", "2024/01/01 08:00:00");
+    setCellContent(model, "F1", "text");
+    setCellContent(model, "F2", "hi");
```

Comment:
Nitpick, but isn't it cleaner/clearer to use `createModelFromGrid` ?

Reply:
yes, but then I have to mount again the spreadsheet component (which is already mounted in the `beforeEach`) (I really don't like `beforeEach` :grimacing: )